js-rattler/src/Version.ts

```diff
@@ -195,4 +195,13 @@ export class Version {
     public withAlpha(): Version {
         return Version.fromNative(this.native.with_alpha());
     }
+
+    /**
+     * Compare 2 versions.
+     *
+     * Returns `-1` if self<other, `0` if self == other, `1` if self > other
```

Comment:
```suggestion
     * Returns `-1` if this<other, `0` if this == other, `1` if this > other
```
I think "this" is the correct terminology.